services/IoT/applications/libraries/java.md

```diff
@@ -311,22 +311,22 @@ The following code sample provides an example implementation of the Status callb
   }
 ```
 
-When the status callback is added to the ApplicationClient, the processDeviceStatus() method is invoked whenever any device is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria. The following code sample shows how you can add the status call back instance into ApplicationClient:
+When the status callback is added to the ApplicationClient, the processDeviceStatus() method is invoked whenever any device is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria. The following code sample shows how you can add the status callback instance to the ApplicationClient:
```

Comment:
"When the status callback is added to the ApplicationClient, the processDeviceStatus() method is invoked whenever any device is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria."
"that matches the criteria" is dangling at the end and it's hard to see what it belongs to. Is it the device the matches the criteria? the connection/disconnection? clarify.